styleguide/screens/Patterns/Calendar/Calendar.js

```diff
@@ -0,0 +1,321 @@
+            }}
+            code={dedent`
+              <DayRange
+                startDate={dayRange.startDate}
+                endDate={dayRange.endDate}
+                selectDate={dayRange.selectDate}
+                onStartDateClick={this.handleStartDateClick}
+                onEndDateClick={this.handleEndDateClick}
+               />
+            `}
+          >
+            <DayRange
+              startDate={dayRange.startDate}
+              endDate={dayRange.endDate}
+              selectDate={dayRange.selectDate}
+              onStartDateClick={this.handleStartDateClick}
+              onEndDateClick={this.handleEndDateClick}
+            />
+          </Specimen>
+
+          <H level={3} className={m.mtLgIi}>Calendar Month</H>
+          <T elm="p" className={m.mtr}>
+            The calendar month should be used to represent the current month,
+            the current day is indicated by type weight.
+          </T>
+
+          <H level={4} className={m.mtLgIi}>Default</H>
+          <Specimen
+            classNames={{
+              root: m.mtr,
+              specimenContainer: m.par,
+            }}
+            code={dedent`
+              <CalendarMonth />
+            `}
+          >
+            <CalendarMonth />
+          </Specimen>
+
+          <H level={4} className={m.mtLgIi}>External Dates</H>
+          <T elm="p" className={m.mtr}>
+            By default the <C>CalendarMonth</C> component hides dates outside of the current month.
+            To overide this behaviour use the <C>showOutOfRange</C> prop show below. The out of
+            range days are indicated with the grey colored type.
+          </T>
+          <Specimen
+            classNames={{
+              root: m.mtr,
+              specimenContainer: m.par,
+            }}
+            code={dedent`
+              <CalendarMonth showOutOfRange />
+            `}
+          >
+            <CalendarMonth showOutOfRange />
+          </Specimen>
+
+          <H level={4} className={m.mtLgIi}>Custom Heading Formatting</H>
+          <T elm="p" className={m.mtr}>
+            The <C>CalendarMonth</C> column heading format can be controlled using the
+            <C>columnHeadingProps</C> and <C>format</C> props.
+          </T>
+          <Specimen
+            classNames={{
+              root: m.mtr,
+              specimenContainer: m.par,
+            }}
+            code={dedent`
+              <CalendarMonth columnHeadingProps={{ format: 'dddd' }} />
+            `}
+          >
+            <CalendarMonth
+              columnHeadingProps={{
+                format: 'dddd',
+              }}
+            />
+          </Specimen>
+
+          <H level={3} className={m.mtLgIi}>Day Picker</H>
+          <T elm="p" className={m.mtr}>
+            Like the calendar month, but allows a day to be selected when clicked or touched this is
+            indicated by a filled square. Using the left and right chevrons you can navigate
```

Comment:
> Like the calendar month, but allows a day to be selected when clicked or touched this is indicated by a filled square.

I don't know if it's me but this sentence sounds weird. 
Couldn't it be something like:

> Like the calendar month, but allows a day to be selected when clicked or touched; the selected day is indicated by a filled square.

?